src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Card/CardPaymentsResource.cs

```diff
@@ -0,0 +1,43 @@
+﻿using SwedbankPay.Sdk.Extensions;
+using SwedbankPay.Sdk.Payments.CardPayments;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class CardPaymentsResource : ResourceBase, ICardResource
+    {
+        public CardPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public async Task<ICardPaymentResponse> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)